UI/temps/dashboard.html

```diff
@@ -85,6 +87,26 @@
             <hr>
             <div><a href="#">A name who reigns without contention</a>.....................................</div>
         </div>
+        <div id="create_group" class="tabcontent">
+            <form>
+                <div id="compose_message">You're creating a New Group</div>
+                <input type="text" placeholder="Name of the group">
+                <hr>
```

Comment:
Instead of <hr> all over your document which makes it untidy & increase your file size unnecessary. I would suggest you instead style the **input** border-bottom to display the bottom line